### PR TITLE
SONAR-6964 Do not display administration sensitive data when opening browser history

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

File: it/it-tests/src/test/java/it/http/HttpHeadersTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import java.io.IOException;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import util.QaOnly;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Category(QaOnly.class)
+public class HttpHeadersTest {
```
Comment (julienlancelot) [inline it/it-tests/src/test/java/it/http/HttpHeadersTest.java]: You should check that image and css are cached

Comment (teryk) [inline it/it-tests/src/test/java/it/http/HttpHeadersTest.java]: done

Comment (julienlancelot) [inline it/it-tests/src/test/java/it/http/HttpHeadersTest.java]: good !

**Comment by julienlancelot** [comment]
LGTM

**Pull request merged**